fem/bilinearform_ext.hpp

```diff
@@ -251,6 +252,33 @@ class PAMixedBilinearFormExtension : public MixedBilinearFormExtension
    void Update();
 };
 
+
+/**
```

Comment:
This comment block could be formatted better (with `@brief`) and maybe clarified.